v2/robotmk/src/session.rs

```diff
@@ -67,3 +80,16 @@ impl CurrentSession {
         }
     }
 }
+
+impl UserSession {
+    fn run(&self, spec: &RunSpec) -> Result<RunOutcome> {
+        run_task(&TaskSpec {
```

Comment:
Should this not be inlined?

Reply:
I decided to leave it as is. We would otherwise have to expose some implementation details from schtasks.rs, so I would rather leave it as is.